src/App.tsx

```diff
@@ -1,25 +1,14 @@
-import React, { useEffect } from 'react';
-import { Routes, Route, useNavigate } from 'react-router-dom';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
 import SigninForm from './components/SigninForm';
 import SignupForm from './components/SignupForm';
 import Navbar from './components/Navbar';
 import Home from './components/Home';
 import ForgotPassword from './components/ForgotPassword';
 import ResetPassword from './components/ResetPassword';
 import ProtectedRoute from './components/ProtectedRoute';
-import { isAuthenticated } from './utility/AuthUtility';
 
 const App: React.FC = () => {
-  const navigate = useNavigate();
```

Comment:
Removed this useEffect because it was checking the user is login or not and send them to the login page for any route.